src/aks-preview/azext_aks_preview/decorator.py

```diff
@@ -1516,6 +1516,15 @@ def get_oidc_issuer_profile(self) -> ManagedClusterOIDCIssuerProfile:
 
         return profile
 
+    def get_crg_id(self) -> str:
```

Comment:
I would appreciate it if you could also write a test case for this "getter" function.

Reply:
Added a test case for it.